src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/MessageContextMenuContent.razor

```diff
@@ -0,0 +1,114 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@using BlazorContextMenu
+@inject FeedbackUI FeedbackUI;
+@inject ChatUI ChatUI;
+
+<Item OnClick="@(() => FeedbackUI.AskFeatureRequestFeedback("reaction", "Reactions"))" CssClass="context-menu-item">
+    <div class="context-menu-content">
```

Comment:
I think we can exrtract a component that define default structure of a menu element.

<div class="context-menu-content">
        <div class="context-menu-icon">
            <i class="@Icon"></i>
        </div>
        <div class="context-menu-text">
            @Text
        </div>
    </div>